### ISSUE TITLE
workflow:dump error if --path is not provided

### ISSUE THREAD

**Issue by klimenttoshkov** [zerodahero/laravel-workflow#66]
```

   TypeError

  League\Flysystem\PathPrefixer::prefixPath(): Argument #1 ($path) must be of type string, null given, called in /Users/sag/Documents/code/zz_api/vendor/laravel/framework/src/Illuminate/Filesystem/FilesystemAdapter.php on line 225

  at vendor/league/flysystem/src/PathPrefixer.php:34
     30▕
     31▕         $this->separator = $separator;
     32▕     }
     33▕
  ➜  34▕     public function prefixPath(string $path): string
     35▕     {
     36▕         return $this->prefix . ltrim($path, '\\/');
     37▕     }
     38▕

      +15 vendor frames
  16  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

if --path="." is provided then dump works fine and outputs to storate/app/.

**Comment by klimenttoshkov** [comment]
Sorry to missing this out, error appeared after updating to Laravel 9.

**Comment by zerodahero** [comment]
Looks like the latest version doesn't treat `null`s the same. I've added in a default that should take care of this in v4.0

**Comment by zerodahero** [comment]
This was addressed in [v4.0.0](https://github.com/zerodahero/laravel-workflow/releases/tag/v4.0.0), please let me know if there's still any issues remaining for this.
